### PR TITLE
Add ltosearch

### PR DESCRIPTION
**Pull request by privatezero**

### CODE REVIEW

File: ingestcollectionchecksum [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+elif [ -f "${1}" ] ; then
+    db_fixity=$(cat "${1}")
+    db_source="${1}"
+    MEDIA_ID=$(basename "${1}" | cut -d'.' -f1)
+    eventType="ingest"
+    _report_to_db
+    _eventoutcome_update
+    _report_fixity_db
+else
+    Echo "Please use a valid directory for input - Exiting" && exit 0
```
Comment (retokromer) [inline ingestcollectionchecksum]: `echo "Please use a valid directory for input. Exiting" && exit 0`

File: ingestschemas [hunk 2]
```diff
@@ -1,22 +1,29 @@
-#!/bin/bash
+#!/usr/bin/env bash
 SCRIPTDIR=$(dirname $(which "${0}"))
 . "${SCRIPTDIR}/mmfunctions"
```
Comment (retokromer) [inline ingestschemas]: I suggest
```
. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ; }
```

File: ingestschemas [hunk 3]
```diff
@@ -1,22 +1,29 @@
...
 
+if [ -z "${PREMIS_NAME}" ] || [ "${PREMIS_DB}" != "Y" ] ; then
+    echo "Please configure database options in mmconfig. Exiting"
+fi
+
 if [ -d "${1}" ] ; then
     cd "${1}"
+    for i in *.schema ; do
+    xmlschema="${i}"
```
Comment (retokromer) [inline ingestschemas]: indent

File: ltosearch [hunk 4]
```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+SCRIPTDIR=$(dirname $(which "${0}"))
+. "${SCRIPTDIR}/mmfunctions"
```
Comment (retokromer) [inline ltosearch]: I suggest
```
. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ; }
```

**Comment by privatezero** [comment]
To date I haven't been using mmfunctions for warnings because I have been experimenting with a broader colour palette than is implemented via mm. (Although the red and green here might be the same).

File: ingestcollectionchecksum [hunk 5]
```diff
@@ -28,6 +28,6 @@ elif [ -f "${1}" ] ; then
     _eventoutcome_update
     _report_fixity_db
 else
-    Echo "Please use a valid directory for input - Exiting" && exit 0
+    Echo "Please use a valid directory for input. Exiting" && exit 1
```
Comment (retokromer) [inline ingestcollectionchecksum]: `echo` not Echo with capital

**Comment by retokromer** [comment]
LGTM

**Comment by dericed** [comment]
maybe needs a readme update at some point, but LGTM

**Comment by retokromer** [comment]
As other scripts are named `SOMETHINGlto` and not `ltoSOMETHING`, I suggest to name this one `searchlto`.

**Comment by privatezero** [comment]
That makes sense to me! @dericed, what do you think? Would engineers here be able to get used to a slightly different command for searching?

**Comment by dericed** [comment]
@privatezero fine with me, searchlto is different enough from ltosearch that distinguishing them would be helpful

**Comment by privatezero** [comment]
Roger that!

**Pull request merged**